pkg/build/builder/daemonless.go

```diff
@@ -306,10 +306,76 @@ func buildDaemonlessImage(sc types.SystemContext, store storage.Store, isolation
 func generateTransientMounts() []string {
 	mounts := []string{}
 	mounts = appendRHSMMount(mounts)
+	mounts = appendETCPKIMount(mounts)
+	mounts = appendRHRepoMount(mounts)
 	mounts = appendCATrustMount(mounts)
 	return mounts
 }
 
+func appendRHRepoMount(mounts []string) []string {
+	st, err := os.Stat("/run/secrets/redhat.repo")
+	if err != nil {
+		// since the presence of /run/secrets/redhat.repo is not a given, we won't log this a V(0) like
+		// we do for /run/secrets/rhsm
+		log.V(4).Infof("Failed to stat /run/secrets/redhat.repo, falling back to the Red Hat yum repository configuration in the build's base image. Error: %v", err)
+		return mounts
+	}
+	if st.IsDir() {
+		// if the file is there, but an unexpected type, then always have log show up via V(0)
+		log.V(4).Infof("Falling back to the Red Hat yum repository configuration in the build's base image: redhat.repo secrets location /run/secrets/redhat.repo is a directory.")
```

Comment:
re: https://github.com/openshift/builder/pull/228#issuecomment-805110570 @adambkaplan yeah this should be unlikely / unexpected to I'll switch to V4

Reply:
sorry V0